tests/utils/shopper.js

```diff
@@ -42,10 +42,19 @@ export const shopper = {
 
 		goToCart: async () => {
 			await shopper.block.goToBlockPage( 'Cart' );
+			await page.waitForSelector(
+				'.wp-block-woocommerce-cart.is-loading',
+				{ hidden: true }
+			);
 		},
 
 		goToCheckout: async () => {
 			await shopper.block.goToBlockPage( 'Checkout' );
+			// Wait for Checkout block to finish loading
```

Comment:
I would add extra details here and for the previous utils that it is good practice to wait for the block to load, otherwise we will get flaky tests.